addons/l10n_tw/data/account.account.template.csv

```diff
@@ -0,0 +1,15 @@
+id,chart_template_id/id,code,name,user_type_id/id,reconcile
```

Comment:
Why are there so few accounts?

Reply:
From the TW community they have another file of account.account.template for other usage, but this file is the most general one. I can ask the FC to check again if there is any additional account.account need to add inside.